OSVectorTileAPI/StylingVTS/Adapting/ol-mapbox-style/LICENSE

```diff
@@ -0,0 +1,22 @@
+Copyright 2016-present ol-mapbox-style contributors
```

Comment:
We've ended up checking in 4 copies of the ol-mapbox-style library and license. I'd move one copy into a higher-up 'lib' directory and adapt the html to point at them

Reply:
Done. In next commit.